Time off tracker WEB/Filters/ExceptionFilter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace TimeOffTracker.WebApi.Filters
+{
+    public class ExceptionFilter : ExceptionFilterAttribute
+    {
+        public override void OnException(ExceptionContext context) 
+        {
+            //Log(logging branch)
```

Comment:
inject ILogger<ExceptionFilter> to constructor and call LogError method - it would log to console, it would ok for now